apps/web/app/utils/confetti.ts

```diff
@@ -0,0 +1,34 @@
+import confetti from 'canvas-confetti'
+
+export default function fireworks() {
+  const duration = 8000 // 8 seconds
+  const animationEnd = Date.now() + duration
+  const defaults = { startVelocity: 30, spread: 360, ticks: 60, zIndex: 0 }
+
+  function randomInRange(min: number, max: number) {
+    return Math.random() * (max - min) + min
+  }
+
+  const interval = setInterval(function () {
+    const timeLeft = animationEnd - Date.now()
+
+    if (timeLeft <= 0) {
+      return clearInterval(interval)
+    }
+
+    const particleCount = 50 * (timeLeft / duration)
+    // since particles fall down, start a bit higher than random
+    confetti(
+      Object.assign({}, defaults, {
+        particleCount,
+        origin: { x: randomInRange(0.1, 0.3), y: Math.random() - 0.2 },
+      })
+    )
+    confetti(
+      Object.assign({}, defaults, {
+        particleCount,
+        origin: { x: randomInRange(0.7, 0.9), y: Math.random() - 0.2 },
+      })
+    )
+  }, 250)
+}
```

Comment:
Hey, I liked the structural changes but I think that poping them from random places on the screen it's too agressive.

Could you please refactor this one to drop "fireworks" from the top?

Reply:
I made a commit to update the duration from 8s to 6.5s, other than that it looks great.

Thank you!